comp/core/secrets/secrets.go

```diff
@@ -69,28 +104,30 @@ func registerSecretOrigin(handle string, origin string, yamlPath []string) {
 
 	if len(yamlPath) != 0 {
 		lastElem := yamlPath[len(yamlPath)-1:]
-		scrubberAddReplacer(lastElem)
+		scrubber.AddStrippedKeys(lastElem)
 	}
 
-	secretOrigin[handle] = append(
-		secretOrigin[handle],
+	r.origin[handle] = append(
+		r.origin[handle],
 		secretContext{
 			origin:   origin,
 			yamlPath: path,
 		})
 }
 
-// Init initializes the command and other options of the secrets package. Since
+// Assign initializes the command and other options of the secrets package. Since
```

Comment:
WDYT about `Configure` instead of `Assign` ?

Reply:
I like it, will make that change.